fw-child/resources/scss/_vendor.scss

```diff
@@ -27,23 +27,33 @@
   flex-wrap: wrap;
   list-style: none;
   padding: 0;
+  margin-bottom: 0;
 
   li {
-    display: flex;
-    margin-bottom: 0.5rem;
-
-    &:not(.share-permalink-wrap) {
-      order: 2;
-
-      .label {
-        display: none;
-      }
+    &:not(:first-child):not(.share-permalink-wrap) {
+      margin-left: 0.75rem;
     }
-
+    
     &.share-permalink-wrap {
-      order: 1;
-      flex: 0 0 100%;
-      width: 100%;
+      display: flex;
```

Comment:
On my larger screen, the "Copy link" button is too close to the "LinkedIn" button and not vertically centered with the other buttons.

(Chrome Windows)

![image](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/159493515/c064ef9f-ddca-4242-b0fb-f1f0f105a520)

I see that the previous developer seems to have somewhat reused the CSS from version 1 (where this button had a different look) for version 2. But in version 2, it's not required that this button has a different style than the other social links, and not required that it be on its own line. So it could simply use the same styling as the other `<li>` of social links.

Reply:
@xfrenette-crim we've unified the styles so the "Copy Link" will have same styling as other sharing buttons.